doc/tutorial/first_steps.md

```diff
@@ -0,0 +1,290 @@
+
+       lxc start third
+
+1. Query more information about each instance with:
+
+       lxc info first
+       lxc info second
+       lxc info third
+       lxc info vm
+
+1. We don't need all of these instances for the remainder of the tutorial, so let's clean some of them up:
+
+   1. Stop the second container:
+
+          lxc stop second
+
+   1. Delete the second container:
+
+          lxc delete second
+
+   1. Delete the third container:
+
+          lxc delete third
+
+      Since this container is running, you get an error message that you must stop it first.
+      Alternatively, you can force-delete it:
+
+          lxc delete third --force
+
+   1. Stop and delete the VM:
+
+          lxc stop vm
+          lxc delete vm
+
+See {ref}`instances-create` and {ref}`instances-manage` for more information.
+
+## Configure instances
+
+There are several limits and configuration options that you can set for your instances.
+See {ref}`instance-options` for an overview.
+
+Let's create another container with some resource limits:
+
+1. Launch a container and limit it to one vCPU and 192 MiB of RAM:
+
+       lxc launch ubuntu:22.04 limited -c limits.cpu=1 -c limits.memory=192MiB
+
+1. Check the current configuration and compare it to the configuration of the first (unlimited) container:
+
+       lxc config show limited
+       lxc config show first
+
+1. Check the amount of free and used memory on the parent system and on the two containers:
+
+       free -m
+       lxc exec first -- free -m
+       lxc exec limited -- free -m
+
+   ```{note}
+   The total amount of memory is identical for the parent system and the first container, because by default, the container inherits the resources from its parent environment.
+   The limited container, on the other hand, has only 192 MiB available.
+   ```
+
+1. Check the number of CPUs available on the parent system and on the two containers:
+
+       nproc
+       lxc exec first -- nproc
+       lxc exec limited -- nproc
+
+   ```{note}
+   Again, the number is identical for the parent system and the first container, but reduced for the limited container.
+   ```
+
+1. You can also update the configuration while your container is running:
+
+   1. Configure a memory limit for your container:
+
+          lxc config set limited limits.memory=128MiB
+
+   1. Check that the configuration has been applied:
+
+          lxc config show limited
+
+   1. Check the amount of memory that is available to the container:
+
+          lxc exec limited -- free -m
+
+      Note that the number has changed.
+
+See {ref}`instances-configure` and {ref}`instance-config` for more information.
+
+## Interact with instances
+
+You can interact with your instances by running commands in them (including an interactive shell) or accessing the files in the instance.
+
+Start by launching an interactive shell in your instance:
+
+1. Run the `bash` command in your container:
+
+       lxc exec first -- bash
```

Comment:
Would it make sense to also introduce the reader to `lxc shell`? I find the syntactic sugar to be quite enjoyable :)

Reply:
I do use it myself too. Although it doesn't work with all distributions, and I remember @stgraber not being so keen on it.

Reply:
We don't use it in the docs so far since it's a hidden command.
And I think it makes sense to consistently stick to one command instead of using a different syntax for running a shell and running any other command.